src/main/eventHandlers/setPreferenceFieldValue.ts

```diff
@@ -11,6 +12,37 @@ const setPreferenceFieldValue = async ({
   info
 }) => {
   const { field, value } = info
+
+  switch (field) {
```

Comment:
Would it be more simple to not have a case-statement just name the events dynamically?
```suggestion
const fieldEventType = `UpdatedField:${field}`
  metricsLogger({
          eventType: fieldEventType
          data: {
            newConcurrentDownloads: value
          }
        })
```

Reply:
I thought about this, but we'd talked about only logging the `newConcurrentDownloads` field when it changes. This would log it every time they close the preferences window.